### ISSUE TITLE
Show ledger's accounts in the Account page

### ISSUE THREAD

**Issue by chevdor** [polkadot-js/apps#4266]
Adding a ledger is done through the *Accounts* page (vs *Address book*).
After adding a ledger, the address shows up under Accounts as expected.

Once this process is finished however (not sure yet whether unplugging and/or refreshing the page is required to see the issue), the ledger account(s) no longer show up under *Accounts* but under *Address book*.

**Comment by jacogr** [comment]
Duplicate of https://github.com/polkadot-js/apps/issues/3788
